apiserver/facades/client/client/status.go

```diff
@@ -1182,9 +1186,23 @@ func (context *statusContext) processApplication(ctx context.Context, applicatio
 	for _, u := range units {
 		appUnits = append(appUnits, u)
 	}
-	displayStatus, err := common.ApplicationDisplayStatus(context.model, application, appUnits)
-	if err == nil {
-		applicationStatus = displayStatus
+
+	// NOTE(jack-w-shaw): If there is an error retrieving the application status,
+	// instead of returning an error we return the application status as unknown.
+	// To me, it is ambiguous if we should do the same retrieving the application
+	// ID. But, but the time the transition to DQLite is completed, this method
+	// will surely take the application ID as a parameter, so we do not need to
+	// worry about this case.
```

Comment:
Status will likely be a small number of calls, so you're right, I don't think it matters long term.